app/utils/CloudWatchClient.scala

```diff
@@ -30,8 +28,8 @@ class CloudWatchClient(stage: String) extends Loggable {
     val request = new PutMetricDataRequest().withNamespace("Typerighter").withMetricData(datum)
 
     try {
-      val result = cloudWatchClient.putMetricData(request)
-      log.info(s"Published metric data: $result")
+      cloudWatchClient.map(_.putMetricData(request))
+      log.info(s"Published metric data")
     } catch {
       case e: Exception =>
         log.error(s"CloudWatch putMetricData exception message: ${e.getMessage}", e)
```

Comment:
Were we going to log a warning here instead, or a message about what data would be sent?

Reply:
I don't think this should be a warning, as the system is behaving as expected – but I definitely think we should log a) that this is a dry run, and b) what it is we were intending to post to CW.